bin/storm.py

```diff
@@ -263,6 +263,34 @@ def upload_credentials(*args):
         jvmtype="-client",
         extrajars=[USER_CONF_DIR, STORM_BIN_DIR])
 
+def blobstore(*args):
+    """Syntax: [storm blobstore [cmd]]
```

Comment:
If `cmd` is not an optional argument, we should remove the square brackets around it.